test/unit/view.js

```diff
@@ -151,101 +247,85 @@ describe('View class', () => {
           },
         };
         view.append(div);
+        assert.calledOnce(view.iframe.document.body.appendChild);
         assert.calledWith(view.iframe.document.body.appendChild, div);
       });
 
-      it('appends to node if no iframe', () => {
+      it('appends to node if iframe does not exist', () => {
         const div = document.createElement('div');
-        const appendChildSpy = sinon.spy(view.component.node, 'appendChild');
+        const appendChildStub = sinon.stub(view.component.node, 'appendChild');
         view.iframe = null;
         view.append(div);
-        assert.calledWith(appendChildSpy, div);
+        assert.calledOnce(appendChildStub);
+        assert.calledWith(appendChildStub, div);
+        appendChildStub.restore();
       });
     });
 
     describe('destroy()', () => {
       it('removes node from parent', () => {
-        const component = new Component({
-          id: 1234,
-        }, {browserFeatures: {}});
-        const view = new View(component);
         view.node = {
           parentNode: {
             removeChild: sinon.spy(),
           },
         };
         view.destroy();
         assert.calledOnce(view.node.parentNode.removeChild);
+        assert.calledWith(view.node.parentNode.removeChild, view.node);
```

Comment:
Same comment as above, can we move out the spy into it's own const, and use that in the assert?